IM_calculation/IM/im_calculation.py

```diff
@@ -257,18 +266,22 @@ def compute_measures_multiprocess(
         )
         i += steps
         array_params = []
+        adv_array_params = []
         for waveform in waveforms:
-            array_params.append(
-                (waveform, ims, comp, period, str_comps, advanced_im_config)
-            )
+            array_params.append((waveform, ims, comp, period, str_comps))
+            adv_array_params.append((waveform, advanced_im_config, output_dir))
+
         result_list = p.map(compute_measure_single, array_params)
+        p.map(compute_adv_measure, adv_array_params)
```

Comment:
Seen this? https://docs.python.org/3/library/multiprocessing.html#multiprocessing.pool.Pool.starmap
Makes the called function cleaner in my opinion